test/benchmarks/performance.test.js

```diff
@@ -298,7 +300,7 @@ describe('performance', function() {
 			},
 			({ ticks, message }) => {
 				console.log(`PERF: large VTree: ${message}`);
-				expect(ticks).to.be.below(2000 * MULTIPLIER);
+				expect(ticks).to.be.below(200 * MULTIPLIER);
```

Comment:
Both 8 and X are well below 2000 ticks here so adjusting to a more reasonable value